tests/test_invenio_logging.py

```diff
@@ -37,6 +37,18 @@
 
 from invenio_logging.fs import InvenioLoggingFs
 from invenio_logging.sentry import InvenioLoggingSentry
+from mock import Mock
+
+
+def test_add_handler(app_for_sentry):
+    ext = InvenioLoggingSentry(app_for_sentry)
```

Comment:
please include docstring with description and move it after `test_init` to keep some logical order of tests